scripts/mobian_installer.sh

```diff
@@ -1,28 +1,69 @@
+echo -n "Do you want to continue? (y/n) --> "
+read yesno < /dev/tty
+if [ "x$yesno" = "xy" ];
+then
+  :
+else
+  echo "Aborting..." & exit 1
+fi
+echo "Installing dependencies..."
+sudo apt update && sudo apt install golang nodejs npm mercurial python qmlscene qml-module-qtwebsockets qml-module-qtmultimedia qml-module-qtwebengine || {
+  echo "Installing dependencies failed." ;
+  exit 1; 
+}
+#The following qml modules have to be installed separately (regex issue).
+sudo apt-get install qml-module-qtquick.controls || {
+  echo "Installing special dependency qml-module-qtquick.controls failed." ;
+  exit 1; 
+}
+sudo apt-get install qml-module-qtquick.dialogs || {
+  echo "Installing special dependency qml-module-qtquick.dialogs failed." ;
+  exit 1; 
+}
+echo "Cloning..."
+go get -d -u github.com/nanu-c/axolotl/ || {
+  echo "Cloning failed" ;
+  exit 1; 
+}
+cd $(go env GOPATH)/src/github.com/nanu-c/axolotl && go mod download || {
+  echo "Downloading (go) failed" ;
+  exit 1; 
+}
+echo "Installing..."
+cd axolotl-web && npm install || {
+  echo "Installing (npm) failed" ;
+  exit 1; 
+}
+#node-sass does not support arm64 so it has to be rebuilt
+echo "Rebuilding of npm-sass..."
+npm rebuild node-sass || {
+  echo "Rebuilding failed" ;
+  exit 1; 
+}
+echo "Building (npm)..."
+npm run build || {
+  echo "Building (npm) failed" ;
+  exit 1; 
+}
 cd .. && mkdir -p build/linux-arm64/axolotl-web
-echo "building (go)..."
-env GOOS=linux GOARCH=arm64 go build -o build/linux-arm64/axolotl .
+echo "Building (go)..."
+env GOOS=linux GOARCH=arm64 go build -o build/linux-arm64/axolotl . || {
+  echo "Building (go) failed" ;
+  exit 1; 
+}
 cp -r axolotl-web/dist build/linux-arm64/axolotl-web && cp -r guis build/linux-arm64
-echo "[Desktop Entry]\nType=Application\nName=Axolotl\nGenericName=Signal Chat Client\nPath=/home/mobian/go/src/github.com/nanu-c/axolotl\nExec=/home/mobian/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl/axolotl\n#Exec=/home/mobian/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl/axolotl -e qt\nIcon=/home/mobian/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl-web/axolotl.png\nTerminal=false\nCategories=Network;Chat;InstantMessaging;Qt;\nStartupWMClass=axolotl" | sudo tee -a /usr/share/applications/axolotl.desktop
+if [ -f /usr/share/applications/axolotl.desktop ];
+then
+  :
+else
+  echo "[Desktop Entry]\nType=Application\nName=Axolotl\nGenericName=Signal Chat Client\nPath=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/\nExec=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl\n#Exec=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl -e qt\nIcon=$HOME/go/src/github.com/nanu-c/axolotl/build/linux-arm64/axolotl-web/dist/axolotl.png\nTerminal=false\nCategories=Network;Chat;InstantMessaging;Qt;\nStartupWMClass=axolotl" | sudo tee -a /usr/share/applications/axolotl.desktop
```

Comment:
echo without -e never writes the new lines \n

anyway the file axolotl.desktop in these new changes is wrong for me

@nanu-c did you test this PR?

Reply:
`echo -e` is indeed not adding new lines via \n in a normal terminal window.
But a simple `echo` used in a sh-script is doing this. Just try and test it.
In the Mobian wiki, there was a copy&paste error in the commands for creating the desktop files, that is true and now fixed. Thanks for pointing out that in this case the `-e` was missing.

Regarding the other thing with your additional axolotl folder in linux-arm64:

In my opinion, the only command that could have caused the additional folder creation, could have been this

`env GOOS=linux GOARCH=arm64 go build -o build/linux-arm64/axolotl .`

and my question is: Did you used it exactly like that? With the dot at the end?

Reply:
yes, following the wiki step by step the .desktop file is wrong

probably it works with your script, I don't know, I didn't test it, but with the wiki the .desktop file doesn't work

did you test the wiki step by step? (without your script)